samples/HelloSignalR/wwwroot/index.html

```diff
@@ -0,0 +1,62 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <script src="https://code.jquery.com/jquery-2.1.4.js" type="text/javascript"></script>
+    <script src="Scripts/jquery.signalR-2.2.0.js" type="text/javascript"></script>
+    <link href="styles/sample.css" rel="stylesheet" />
+    <title>ASOS SignalR Sample</title>
+    <script type="text/javascript">
+        'use strict';
+        $(document).ready(function () {
+            let connectionBase = "http://localhost:5000";
+            let connectionPath = `${connectionBase}/signalr`;
```

Comment:
Looks like some new ECMAScript goodness is heavily used on this file :smile: 
Can we safely assume it will work on most browsers or is it worth using a transpiler thingy?